core/src/trezor/ui/layouts/tr/__init__.py

```diff
@@ -399,27 +399,35 @@ async def confirm_reset_device(
         )
     )
 
+    if recovery:
+        await confirm_action(
+            "recover_device",
+            title,
+            description="It is safe to eject your Trezor anytime and continue later.",
+            verb="CONTINUE",
+            br_code=ButtonRequestType.ProtectCall,
+        )
+
 
-# TODO cleanup @ redesign
 async def confirm_backup() -> bool:
```

Comment:
missed this in some previous review:
**confirm**_something should raise when the user cancels.

please rename here and in TT to `ask_backup` or `prompt_backup`

Reply:
Done in [546199b](https://github.com/trezor/trezor-firmware/pull/3135/commits/546199beaf948ff9e4298cebec355e1b1e5cd959)